ci/ciCheck.js

```diff
@@ -62,7 +62,7 @@ async function checkNodeVersions() {
     ignoreReleasedVersions: [
       '<12.0.0', // These versions have reached their end-of-life support date
       '>=13.0.0 <14.0.0', // These versions have reached their end-of-life support date
-      '>=16.0.0', // This version has not been officially released yet
+      '>=15.0.0 <16.0.0', // These versions have reached their end-of-life support date
```

Comment:
Could you please add a line 

```
'>=17.0.0', // These versions are not officially supported yet
```

We will have to add Node 17 CI support in a separate issue / PR.

Reply:
Sure!